lib/components/narrative/narrative-itineraries.js

```diff
@@ -51,7 +52,13 @@ class NarrativeItineraries extends Component {
 
   static contextType = ComponentContext
 
-  state = {}
```

Comment:
This isn't the right spot for this comment, but I think `NarrativeItinerary#_onHeaderClick` might need to invoke the `setItineraryView` action:

https://github.com/opentripplanner/otp-react-redux/blob/ef9554794272cbac5b73c75772e39dd69e674315/lib/components/narrative/narrative-itinerary.js#L20-L29

Reply:
Updated in https://github.com/opentripplanner/otp-react-redux/pull/339/commits/19f056929e4f8ee17fadc8f1ac3c795ee29f3878.